docs/en/writing-running-appium/web/chromedriver.md

```diff
@@ -137,3 +141,8 @@ CHROMEDRIVER_CDNURL=http://npm.taobao.org/mirrors/chromedriver npm install appiu
 
 It may also be necessary to adjust network proxy and firewall settings to allow
 the download to occur.
+
+### W3C support
+Chrome Driver hasn't followed the W3C standard yet. If you encounter proxy command error like [this issue](https://github.com/appium/python-client/issues/234), please update your chrome driver version.
```

Comment:
Chromedriver is a singe word, capitalized at the beginning. Might as well stay consistent.